terraform/variables.tf

```diff
@@ -0,0 +1,25 @@
+variable "stack" {
+    type = string
+    default = "cyberworld"
+}
+
+variable "env" {
+    type = string
+    default = "dev"
+}
+
+# TODO: Make these pull from terraform in the future
+variable "certificate_arn" {
+    type = string
```

Comment:
the certificate and the domain are the only resources that are not expressed in terraform code. i'm making tickets to roll these in as well.